chromadb/cli/cli.py

```diff
@@ -1,8 +1,29 @@
+import time
+
+import requests
 import typer
 import uvicorn
 import os
 import webbrowser
 
+import chromadb
+from chromadb.utils import system_info_utils
+
+try:
+    from rich import print as rprint
+except ImportError:
+    rprint = typer.echo
+
+
+def format_size(size: int) -> str:
+    try:
+        from humanfriendly import format_size as hf_format_size
```

Comment:
Why not just make this a dep?

Reply:
it's purely optional but if it is present, it will format something hard to read by humans, like memory size 77561856 to 77.79 MB. We can remove it altogether and just replace it with a simple function that does the conversion since there aren't that many places where we have large numbers to deal with at the moment.

Reply:
```python
ef format_size(size_in_bytes):
    units = ["B", "KB", "MB", "GB", "TB"]
    unit_index = 0

    while size_in_bytes > 1024 and unit_index < 4:
        size_in_bytes /= 1024.0
        unit_index += 1

    return f"{size_in_bytes:.2f} {units[unit_index]}"

# Test
print(format_size(1234567890))  # Outputs: "1.15 GB"
```

Reply:
Added the function directly so we no longer need to have humanfriendly to pretty print large numbers.